packages/contracts/package.json

```diff
@@ -22,7 +22,7 @@
     "tsx": "^3.12.2"
   },
   "scripts": {
-    "test": "tsx ./scripts/runner/runner.ts",
+    "test": "forge test",
```

Comment:
I've noticed issues in the past with conflicting arguments/parameters for shell to pass to yarn -> forge. Are you sure this is stable? Perhaps its better to do: 

```suggestion
    "forge:test": "forge test",
```